DateRangeBox: add code snippets and extended description #5043 (#5111)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxDateRangeBox/1 Configuration/startDateText.md

---
---
##### shortDescription

> The read-only property that holds the text displayed by the end date input field. 

Reviewer: ```suggestion
Returns the text displayed by the start date input field. 
```
